src/output.cpp

```diff
@@ -527,8 +527,8 @@ std::pair<double, double>
 mean_stdev(const double* x, int n)
 {
   double mean = x[static_cast<int>(TallyResult::SUM)] / n;
-  double stdev = n > 1 ? std::sqrt((x[static_cast<int>(TallyResult::SUM_SQ)]/n
-    - mean*mean)/(n - 1)) : 0.0;
+  double stdev = n > 1 ? std::sqrt(std::max(0.0, (
+    x[static_cast<int>(TallyResult::SUM_SQ)]/n - mean*mean)/(n - 1))) : 0.0;
```

Comment:
Can the variance be negative?

Reply:
I ran into this when using the analog heating estimator. In the case of a monoenergetic photon source, every particle scores exactly the energy it started with, so the variance is expected to be zero. However, due to limited precision and roundoff errors, sometimes 0 ends up being some very small negative number, which causes problems here.

Reply:
gotcha, makes sense!